extension-requests/script.js

```diff
@@ -538,8 +552,25 @@ async function createExtensionCard(data, dev) {
       id: 'title',
       name: 'title',
       value: data.title,
+      'data-testid': 'title-text-input',
```

Comment:
Nameing? 
data-test-id?

Reply:
![image](https://github.com/user-attachments/assets/09416c65-3da9-40a5-bebb-f35734caebf0)

this name used previously in other elements so I used the same in mine changes too.

Reply:
Can you do a quick google and verity it once please which is preferred?

Reply:
![image](https://github.com/user-attachments/assets/0a26576f-22ba-4d0b-a337-2cd0f2015346)

@prakashchoudhary07 `data-testid` is preferred.